src/transformers/models/encoder_decoder/modeling_flax_encoder_decoder.py

```diff
@@ -670,6 +666,11 @@ def __call__(
             batch_size, sequence_length = input_ids.shape
             position_ids = jnp.broadcast_to(jnp.arange(sequence_length)[None, :], (batch_size, sequence_length))
 
+        # prepare decoder inputs
+        if decoder_input_ids is None:
+            raise ValueError(
+                "For sequence to sequence training, `decoder_position_ids` must be specified as an input argument."
```

Comment:
maybe write it this way
```suggestion
                "`decoder_input_ids` can not be `None`. For sequence to sequence training, `decoder_input_ids` must be specified as an input argument."
```